### ISSUE TITLE
Host Docker image on Pinterest Docker account

### ISSUE THREAD

**Issue by derekargueta** [pinterest/rocksplicator#326]
We should probably move `rajathprasad/rocksplicator-build:latest` to something at the organization level like `pinterest/rocksplicator-build:latest`

**Comment by newpoo** [comment]
Agreed. What would be the process to upload an image to pinterest account
on dockerhub?

**Comment by derekargueta** [comment]
@newpoo I don't know, I started an internal email thread with you cc'd.
